NineChronicles.Headless/ActionEvaluationPublisher.cs

```diff
@@ -345,6 +354,13 @@ NodeStatusRenderer nodeStatusRenderer
                                 // FIXME add logger as property
                                 Log.Error(e, "[{ClientAddress}] Skip broadcasting render due to the unexpected exception", _clientAddress);
                             }
+
+                            if (SentryTraces.TryRemove(ev.TxId?.ToString() ?? "", out var sentryTrace))
```

Comment:
Can we ignore the process when `ev.TxId` is null?  (if so, some tweak on this guard would be better)